scripts/docker/build/linux/tools/install.sh

```diff
@@ -37,19 +37,5 @@ make install_dev
 cd /tmp
 rm -rf openssl-$OPENSSLVERSION.tar.gz openssl-$OPENSSLVERSION
 
-# Compile openldap library:
-export OPENLDAPVERSION=2.6.6
```

Comment:
Note: I originally removed this section in my PR to remove LDAP support.
Max wanted the OpenLDAP section to remain though. I can't recall why exactly, but I think it would be good if you could check with him.

Reply:
@neunhoef we can restore this in future in case of a need. Currently that gives zero effort since this library is unused.

Reply:
We want to be able to build 3.11 with this build image. Therefore, I would leave the openldap stuff in for now.